benchmarks/src/main/java/io/druid/benchmark/query/SearchBenchmark.java

```diff
@@ -146,56 +156,156 @@ public int columnCacheSizeBytes()
-      resultNum = (int) (100000 * 0.1);
-      step = 100000 / resultNum;
-      for (int i = 0; i < 100001 && dimHyperUniqueFilterVals.size() < resultNum; i += step) {
-        dimHyperUniqueFilterVals.add(String.valueOf(i));
+      @Override
+      public String apply(String value)
+      {
+        return String.valueOf(Long.parseLong(value) + 1);
       }
 
-      final List<DimFilter> dimFilters = Lists.newArrayList();
-      dimFilters.add(new InDimFilter("dimUniform", dimUniformFilterVals, null));
-      dimFilters.add(new InDimFilter("dimHyperUnique", dimHyperUniqueFilterVals, null));
+      @Override
+      public boolean preservesOrdering()
+      {
+        return false;
+      }
 
-      Druids.SearchQueryBuilder queryBuilderB =
-          Druids.newSearchQueryBuilder()
-                .dataSource("blah")
-                .granularity(QueryGranularities.ALL)
-                .intervals(intervalSpec)
-                .query("")
-                .dimensions(Lists.newArrayList("dimUniform", "dimHyperUnique"))
-                .filters(new AndDimFilter(dimFilters));
+      @Override
+      public ExtractionType getExtractionType()
+      {
+        return ExtractionType.ONE_TO_ONE;
+      }
+    }, null));
+    dimFilters.add(new InDimFilter(dimName, dimUniformFilterVals, new LowerExtractionFn(null)));
+    dimFilters.add(new InDimFilter(dimName, dimUniformFilterVals, new UpperExtractionFn(null)));
+    dimFilters.add(new InDimFilter(dimName, dimUniformFilterVals, new SubstringDimExtractionFn(1, 3)));
+
+    return Druids.newSearchQueryBuilder()
+                 .dataSource("blah")
+                 .granularity(QueryGranularities.ALL)
+                 .intervals(intervalSpec)
+                 .query("")
+                 .dimensions(Lists.newArrayList("dimUniform"))
+                 .filters(new AndDimFilter(dimFilters));
+  }
+
+  private static SearchQueryBuilder basicD(final BenchmarkSchemaInfo basicSchema)
+  {
+    final QuerySegmentSpec intervalSpec = new MultipleIntervalSegmentSpec(Arrays.asList(basicSchema.getDataInterval()));
 
-      basicQueries.put("B", queryBuilderB);
+    final List<String> dimUniformFilterVals = Lists.newArrayList();
+    final int resultNum = (int) (100000 * 0.1);
+    final int step = 100000 / resultNum;
+    for (int i = 1; i < 100001 && dimUniformFilterVals.size() < resultNum; i += step) {
+      dimUniformFilterVals.add(String.valueOf(i));
     }
 
-    SCHEMA_QUERY_MAP.put("basic", basicQueries);
+    final String dimName = "dimUniform";
+    final List<DimFilter> dimFilters = Lists.newArrayList();
+    dimFilters.add(new InDimFilter(dimName, dimUniformFilterVals, null));
+    dimFilters.add(new SelectorDimFilter(dimName, "3", null));
+    dimFilters.add(new BoundDimFilter(dimName, "100", "10000", true, true, true, null, null));
+    dimFilters.add(new InDimFilter(dimName, dimUniformFilterVals, null));
+    dimFilters.add(new InDimFilter(dimName, dimUniformFilterVals, null));
```

Comment:
@jihoonson did you intentionally add three identical lines in a row?

Reply:
It looks a mistake. Thanks for finding.